notification/messagestatus.go

```diff
@@ -19,6 +19,9 @@ type MessageStatus struct {
 	// Details can contain any additional information about the State (e.g. "ringing", "no-answer" etc..).
 	Details string
 
+	//LastStatus labels information given in details for quick overall (failed, delivered, etc..).
+	LastStatus MessageLastStatus
+
```

Comment:
This is already represented by `State` (if not as granular).

Rather than adding a new field, let's update `MessageState`

We can add `MessageStatePending` and change `MessageStateActive` to `MessageStateSending` to make it a bit more clear. Pending probably shouldn't result in `Active` as it does today anyhow. Don't forget to update the switch statement

The DB/Store isn't the only place generating `MessageStatus` so we'd have to update all sites to be consistent, however since it's only used for display purposes we can just make `State` do what we want and then add the text in the GraphQL layer.